README.md

```diff
@@ -1,17 +1,31 @@
 # eLife API Validator for Python
--
+================================
+This library provides a validator for the [eLife Sciences API](https://github.com/elifesciences/api-raml).
+
+It validates HTTP responses to make sure that they match the schema specification for that media type.
 
 Dependencies
 ------------
 
--
+* Python 3.5 or greater
 
 Installation
 ------------
 
--
+`python setup.py install`
```

Comment:
I would have thought `pip install ...`?

Reply:
The issue with uploading to PyPi is that this project requires the static JSON schema files as well as the source. If the schemas are updated we would need to bump the version number each time as PyPi does not let you amend or add files to current versions as you could break a working version and re submit it.

Reply:
Can we skip PyPi with this (it seems we are doing it):
https://github.com/elifesciences/profiles/blob/develop/requirements.txt#L2
Or is this a mis-use that leads us to having to do this every time:
https://github.com/elifesciences/profiles/blob/develop/install.sh#L14-L16
An important part would be that if someone checks out another project like `profiles`, the version that comes in for `api-validator-python` is deterministic, always the same commit.

Reply:
Yes that is the way I envisioned using it. As this is only be used in the profiles tests at present I don't see any issues removing and installing. Though I need to run some tests in the near future to see If this kind of thing (having to uninstall and reinstall python repos) can just be handled with pip flags.